cmd/subctl/join.go

```diff
@@ -293,8 +289,6 @@ func getNetworkDetails(clientProducer client.Producer, status reporter.Interface
 		status.Warning("No network details discovered")
 	}
 
-	status.End()
```

Comment:
This actually changes the output behavior, ie the details would now be printed on L299 before the status message which we don't want.

Reply:
Ah yes, good point. I actually thought about using `func{}()` constructs, JavaScript-style, but that ends up too cumbersome for a general pattern.